### PR TITLE
tz 0.6.2

### PR DESCRIPTION
**Pull request by mitchblank**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

There have been previous attempts to bump this formula (#114358 #115039) but they were abandoned, maybe due to CI flakiness(?)

Until a few minutes ago the status of 0.6.2 was ambiguous since it hadn't been marked as a "release" on github.  That is fixed now though https://github.com/oz/tz/issues/34 so let's try again.

### CODE REVIEW

**Comment by mitchblank** [comment]
Well, CI failed in the same way again:
```
  Error: ==> Testing tz
  Error: tz: failed
  An exception occurred within a child process:
    Minitest::Assertion: Expected /\e\[/ to match "".
```
This formula has always been hard to test...
```ruby
test do
    # Bubbletea-based apps are hard to test even under PTY.spawn (or via
    # expect) because they rely on vt100-like answerback support, such as
    # "<ESC>[6n" to report the cursor position.  For now we just run
    # the command for a second and see that it tried to send some ANSI out of it.
    require "pty"
    r, _, pid = PTY.spawn "#{bin}/tz", "-q"
    sleep 1
    Process.kill("TERM", pid)
```
...but apparently something changed in the CI environment so that even this pty-based hack stopped working?  Needless to say, it works when running `brew test tz` manually.

In any case, I don't think the behavior of the package has actually changed -- something in the CI is different from when 0.6.1 was accepted in 2021 (#87243)  Any CI wizards know what could have changed?

**Comment by chenrui333** [comment]
```
    Minitest::Assertion: Expected /\e\[/ to match "".
```

**Comment by mitchblank** [comment]
> ```
>     Minitest::Assertion: Expected /\e\[/ to match "".
> ```

Yes, but the question is *why*

As described in the existing comment, I wasn't able to create much of a meaningful test for this formula because its deep dependence on talking to a vt100-like tty.  (This is true even if `$TERM` is set to something else, btw)  So the best I was able to come up with was to at least check that it starts up and starts to try sending some ANSI control codes at us.

This still works if you run `brew test` manually (at least on my Ventura/Intel system) but for some reason under CI it seems the pty isn't capturing any output.

I've definitely found in the past that pty-dependent tests are more finicky when run in the CI.  I don't know why that is, though (maybe some permission issue or ulimit that prevents the pty allocation inside the CI environment?)   But this particular test is one that used to pass CI but now seems to be failing for some reason I don't understand.

File: Formula/tz.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Tz < Formula
   desc "CLI time zone visualizer"
   homepage "https://github.com/oz/tz"
-  url "https://github.com/oz/tz/archive/refs/tags/v0.6.1.tar.gz"
-  sha256 "96be9896deb2e27452d2dfde4d7121a1bd6fa1f8eec3d3b8f374b4d55626043d"
+  url "https://github.com/oz/tz/archive/refs/tags/v0.6.2.tar.gz"
+  sha256 "778373cf9b0d4e4bec6b9f101da070b9b25498374c74fbd11f148cbd4257d7ad"
```
Comment (chenrui333) [inline Formula/tz.rb:5]: The sha actually got updated 😮‍💨

Comment (mitchblank) [reply Formula/tz.rb]: Yeah, I think it was respun because I filed https://github.com/oz/tz/issues/35

**Comment by chenrui333** [comment]
I dont know why the test was written as that complicated, just simplify a bit and passed on the action runner.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@mitchblank should be good now

**Comment by mitchblank** [comment]
Ah OK.  I think the `--list` option was added at some point.  Originally `-v` was the only option that didn't require a pty to run and people get sniffy if running `program --version` is your only unit test

**Pull request closed without merging**